specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimnamedvalues.json

```diff
@@ -445,6 +445,57 @@
       }
     }
   },
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/namedValues/{namedValueId}/refreshSecret": {
+    "post": {
+      "tags": [
+        "NamedValue"
+      ],
+      "operationId": "NamedValue_RefreshSecret",
+      "description": "Refresh the secret of the named value specified by its identifier.",
+      "x-ms-examples": {
+        "ApiManagementRefreshNamedValue": {
+          "$ref": "./examples/ApiManagementRefreshNamedValue.json"
+        }
+      },
+      "parameters": [
+        {
+          "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+        },
+        {
+          "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+        },
+        {
+          "$ref": "./apimanagement.json#/parameters/NamedValueIdParameter"
+        },
+        {
+          "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+        },
+        {
+          "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+        }
+      ],
+      "responses": {
+        "200": {
```

Comment:
>200 [](start = 9, length = 3)

This is a long running operation so look at `PUT`. you will need to add
`"x-ms-long-running-operation": true,`